components/openindiana/time-slider/Makefile

```diff
@@ -66,6 +68,11 @@ install: $(INSTALL_32)
 
 download:: $(SOURCE_DIR)/.downloaded
 
+REQUIRED_PACKAGES += shell/ksh93
+REQUIRED_PACKAGES += library/glib2
+PYTHON_REQUIRED_PACKAGES += runtime/python
+# Auto-generated dependencies
```

Comment:
You have added this twice - one time automatically (below) and one time manually.